### PR TITLE
edits from AD review 

### PR DESCRIPTION
**Pull request by b---c**
edits addressing AD review https://mailarchive.ietf.org/arch/msg/oauth/gd9k8wF2h9mBCoqORaD83vbRFl0/

### CODE REVIEW

File: main.md [hunk 1]
```diff
@@ -323,7 +324,7 @@ A DPoP proof is a JWT ([@!RFC7519]) that is signed (using JSON Web Signature (JW
 JOSE header of a DPoP JWT MUST contain at least the following parameters:
 
  * `typ`: with value `dpop+jwt`, which explicitly types the DPoP proof JWT as recommended in [@RFC8725, section 3.11].
- * `alg`: a digital signature algorithm identifier such as per [@!RFC7518].
+ * `alg`: a JWS algorithm identifier, from the registered algorithms [@IANA.JOSE.ALGS], of an asymmetric digital signature.
```
Comment (danielfett) [inline main.md:327]: Maybe like this?
```suggestion
 * `alg`: an identifier for a JWS asymmetric digital signature algorithm from [@IANA.JOSE.ALGS].
```

File: main.md [hunk 2]
```diff
@@ -1186,23 +1188,24 @@ DPoP proof JWTs with unnecessarily large `jti` values or store only a hash there
...
+created in this way yield the same result even in the face of arbitrarily
+large clock skews.
 
 Server-provided nonces are an effective means for further reducing the chances for successful DPoP proof replay.
 Unlike cryptographic nonces, it is acceptable for clients to use the same 
 `nonce` multiple times, and for the server to accept the same nonce multiple
-times. If `jti` is enforced unique for the lifetime of the `nonce`, there is no
-additional risk of token replay.
+times. As long as the `jti` value is tracked for the lifetime of the `nonce`, there
+is no additional risk of token replay.
```
Comment (danielfett) [inline main.md:1202]: I'm not sure if 'tracked' is enough here?

File: main.md [hunk 3]
```diff
@@ -1186,23 +1188,24 @@ DPoP proof JWTs with unnecessarily large `jti` values or store only a hash there
...
 
 ## DPoP Proof Pre-Generation {#Pre-Generation}
 
-An attacker in control of the client can pre-generate DPoP proofs for use
-arbitrarily far into the future by choosing the `iat` value in the
-DPoP proof to be signed by the proof-of-possession key.
+An attacker in control of the client can pre-generate DPoP proofs for
+specific endpoints to use arbitrarily far into the future by choosing the
```
Comment (danielfett) [inline main.md:1207]: There's some ambiguity here which we can avoid by removing the "to use" part.
```suggestion
specific endpoints arbitrarily far into the future by choosing the
```

File: main.md [hunk 4]
```diff
@@ -1274,7 +1277,7 @@ layer of defense against cross-site scripting.
 
 ## Signed JWT Swapping
 
-Servers accepting signed DPoP proof JWTs MUST check the `typ` field in the
+Servers accepting signed DPoP proof JWTs MUST check the `typ` field is `dpop+jwt` in the
```
Comment (danielfett) [inline main.md:1280]:
```suggestion
Servers accepting signed DPoP proof JWTs MUST verify that the `typ` field is `dpop+jwt` in the
```

**Review by danielfett — CHANGES_REQUESTED** [review]
Thanks for updating the draft! I have some minor comments, but overall this looks good to me.

File: main.md (hunk 1 above)
Comment (b---c) [reply main.md:327]: wfm

File: main.md (hunk 2 above)
Comment (danielfett) [reply main.md:1202]: Sounds good!

**Comment by b---c** [comment]
>

**Comment by b---c** [comment]
Thanks for the minor comments/suggestions. I believe all have been incorporated.

**Review by danielfett — APPROVED** [review]

**Pull request merged**